src/main/java/no/nav/foreldrepenger/selvbetjening/http/filters/HeadersToMDCFilterBean.java

```diff
@@ -56,7 +58,9 @@ private void putValues(HttpServletRequest request, String uri) {
             toMDC(NAV_USER_ID, Optional.ofNullable(tokenUtil.getSubject()).map(StringUtil::partialMask).orElse("Uautentisert"));
             toMDC(NAV_AUTH_LEVEL, Optional.ofNullable(tokenUtil.getLevel()).map(AuthenticationLevel::name).orElse(AuthenticationLevel.NONE.name()));
             toMDC(NAV_CALL_ID, request.getHeader(NAV_CALL_ID), generator.create());
-
+            if (tokenUtil.erAutentisert()) {
```

Comment:
Bør logge fullt endepunkt også for å kunne skille interessante kall fra hverandre. Kan droppe callid fra message ettersom bruksmønsteret vel blir å finne feltet i kibana og klikke på feltet for å pinne verdien og ta med over fra tjenestekall til applogg.
Foreslår noe så enkelt som:
FNR – PATH

Reply:
Får man den ikke gjennom traefikk Steffen? Rett meg om jeg tar feil.

Reply:
Stemmer at man får path (+ masse data om nettleser osv) via reverse proxy innslagene. Men på stående fot usikker på om vi får callId.. kan sjekke.